.github/workflows/test.yml

```diff
@@ -1,4 +1,4 @@
-name: Run tests
+name: Tests
```

Comment:
Please don't rename the action. This will create two jobs listed until the logs expire with no appreciable benefit other than preference.

Reply:
I've reverted this too @jpogran. All good.

> This will create two jobs listed until the logs expire

But just an FYI, this is no longer the case - GitHub have fixed this issue - the jobs are now grouped based on the workflow filename - and the left hand job filter will reflect that too - only showing the job `name:` that is currently set within the yaml workflow definition.

Agree - this _was_ annoying in the past.

Reply:
Good to know!